herbert_core/utilities/maths/sawtooth_iarray.m

```diff
@@ -1,38 +1,47 @@
 function ivout = sawtooth_iarray (n)
-% Create array of successive runs [1,2,3..n(1),1,2,3...n(2),...]'
+% Create column vector [1;2;3..n(1); 1;2;3...n(2); ...] from integer array n
 %
 %   >> ivout = sawtooth_iarray (n)
 %
+% This is a 'sawtooth' array as each element n(i) defines a right angle triangle
+% with length n and values 1,2,3,...n(i).
```

Comment:
this right angle statement baffles me. Sawtooth (other array descriptions of this type say jagged, but that's fine) does not seem compatible with this triangle picture

Reply:
I was hoping the text would help, but probably the description on line 2 i.e. it is the array
[1;2;3..n(1); 1;2;3...n(2); ...]
is more useful. I've deleted the description. on lines 6 and 7